README.md

```diff
@@ -43,6 +43,8 @@ composer require owncloud/ocis-php-sdk
 Create an Ocis object using the service Url and an access token:
 ```php
 $ocis = new Ocis('https://example.ocis.com', $accessToken);
+// for using education endpoint use education api token
+$ocis = new Ocis('https://education.ocis.com', null, [], $educationAccessToken);
 ```
```

Comment:
I think this is a bit confusing.
On top we are talking about an `access token`
In between there is an `education api token` but the variable is called `$educationAccessToken`
And then afterwards we tell people how to get an access token.

I think you will have to elaborate about
1. that there are two different tokens for different API endpoints
2. that at least one needs to be provided
3. where to get an education access token
4. which token is needed for which functions
5. which token the chapter below is talking about